latest/index.bs

```diff
@@ -93,6 +93,24 @@ of bioimaging data, whether during acquisition or sharing in the cloud.
 Note: The following text makes use of OME-Zarr [[ome-zarr-py]], the current prototype implementation,
 for all examples.
 
+Document conventions
+--------------------
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”,
+“RECOMMENDED”, “MAY”, and “OPTIONAL” are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+<p>
+<dfn>Transitional</dfn> metadata is added to the specification with the
```

Comment:
Does this need it's own `Transitional metadata` heading (instead of being under `Document conventions`?

Reply:
Ah - I see that's got it's own section below. ok